firebase-appdistribution/src/main/res/layout/activity_feedback.xml

```diff
@@ -100,25 +100,35 @@
                 android:background="@drawable/card_background"
                 android:orientation="vertical">
 
-                <CheckBox
-                    android:id="@+id/screenshotCheckBox"
-                    android:layout_width="wrap_content"
+                <RelativeLayout
+                    android:layout_width="match_parent"
                     android:layout_height="wrap_content"
-                    android:text="@string/screenshot" />
+                    android:layout_marginTop="5dp"
+                    android:layout_marginBottom="5dp">
```

Comment:
Curious what Tars thinks, but this looks a little cramped to me. I'd probably increase the bottom margin a bit here, maybe to `10dp`?